tests/ES512TokenTest.php

```diff
@@ -66,31 +66,31 @@ public function createConfiguration(): void
     #[PHPUnit\Test]
     public function builderShouldRaiseExceptionWhenKeyIsInvalid(): void
     {
-        $builder = $this->config->builder();
+        $builder = $this->config->builder()
+            ->identifiedBy('1')
+            ->permittedFor('https://client.abc.com')
+            ->issuedBy('https://api.abc.com')
+            ->withClaim('user', ['name' => 'testing', 'email' => 'testing@abc.com']);
 
         $this->expectException(InvalidKeyProvided::class);
         $this->expectExceptionMessage('It was not possible to parse your key, reason:');
 
-        $builder->identifiedBy('1')
-            ->permittedFor('https://client.abc.com')
-            ->issuedBy('https://api.abc.com')
-            ->withClaim('user', ['name' => 'testing', 'email' => 'testing@abc.com'])
-            ->getToken($this->config->signer(), InMemory::plainText('testing'));
+        $void = $builder->getToken($this->config->signer(), InMemory::plainText('testing'));
```

Comment:
Minor improvement here: all the logics that don't raise the exception yet should go before the `expectException`